platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/SyncGossip.java

```diff
@@ -336,59 +375,19 @@ private void buildSyncProtocolThreads(
                     .setOtherNodeId(otherId)
                     .setThreadName("SyncProtocolWith" + otherId)
                     .setHangingThreadPeriod(hangingThreadDuration)
-                    .setWork(new NegotiatorThread(
+                    .setWork(new ProtocolNegotiatorThread(
                             connectionManagers.getManager(otherId, topology.shouldConnectTo(otherId)),
                             syncConfig.syncSleepAfterFailedNegotiation(),
                             List.of(
                                     new VersionCompareHandshake(
                                             appVersion, !protocolConfig.tolerateMismatchedVersion()),
                                     new HashCompareHandshake(epochHash, !protocolConfig.tolerateMismatchedEpochHash())),
```

Comment:
and the handshake protocols are going to be done in a subsequent PR?

Reply:
updated